fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -0,0 +1,125 @@
+
+        final Namespaces ns = new Namespaces("rdf", RdfNamespaces.RDF);
+        ns.add("fedora", RdfNamespaces.REPOSITORY);
+
+        //boolean writeBinaries = Boolean.parseBoolean(
+         //       getContext().resolvePropertyPlaceholders("{{serialization.includeBinaries}}"));
+
+        /**
+         * A generic error handler (specific to this RouteBuilder)
+         */
+        onException(Exception.class)
+                .maximumRedeliveries("{{error.maxRedeliveries}}")
+                .log("Index Routing Error: ${routeId}");
+
+        /**
+         * Handle Serialization Events
+         */
+        from("{{input.stream}}")
+            .routeId("FcrepoSerialization")
+            .filter(not(or(header(FCREPO_IDENTIFIER).startsWith(simple("{{audit.container}}/")),
+                    header(FCREPO_IDENTIFIER).isEqualTo(simple("{{audit.container}}")))))
+            .choice()
+                .when(header(EVENT_TYPE).isEqualTo(RdfNamespaces.REPOSITORY + "NODE_REMOVED"))
+                    .multicast().to("direct:delete.metadata", "direct:delete.binary").endChoice()
+                .otherwise()
+                    .multicast().to("direct:metadata", "direct:binary");
+
+        from("{{serialization.stream}}")
+            .routeId("FcrepoReSerialization")
+            .filter(not(or(header(FCREPO_IDENTIFIER).startsWith(simple("{{audit.container}}/")),
+                    header(FCREPO_IDENTIFIER).isEqualTo(simple("{{audit.container}}")))))
+            .multicast().to("direct:metadata", "direct:binary");
+
+        from("direct:metadata")
+            .routeId("FcrepoSerializationMetadataUpdater")
+            .to("fcrepo:{{fcrepo.baseUrl}}?accept={{serialization.mimeType}}")
+            .log(INFO, LOGGER,
+                    "Serializing object ${headers[CamelFcrepoIdentifier]}")
+            .setHeader(FILE_NAME)
+                .simple("${headers[CamelFcrepoIdentifier]}.{{serialization.extension}}")
+            .log(DEBUG, LOGGER, "filename is ${headers[CamelFileName]}")
+            .to("{{serialization.descriptions}}");
+
+        from("direct:binary")
+            .routeId("FcrepoSerializationBinaryUpdater")
+            .filter().simple("{{serialization.includeBinaries}} == 'true'")
+            .to("fcrepo:{{fcrepo.baseUrl}}")
+            .filter().xpath(isBinaryResourceXPath, ns)
+            .log(INFO, LOGGER,
+                    "Writing binary ${headers[CamelFcrepoIdentifier]}")
+            .to("fcrepo:{{fcrepo.baseUrl}}?metadata=false")
+            .setHeader(FILE_NAME).header(FCREPO_IDENTIFIER)
+            .log(DEBUG, LOGGER, "header filename is: ${headers[CamelFileName]}")
+            .to("{{serialization.binaries}}");
+
+        from("direct:delete.metadata")
+            .routeId("FcrepoSerializationMetadataDeleter")
+            .setHeader(EXEC_COMMAND_ARGS).simple(
+                    "{{serialization.descriptions}}/${headers[CamelFcrepoIdentifier]}{{serialization.extension}}")
```

Comment:
No need for the "/" character here, since all fcrepo identifiers will be prefixed with a "/".